### ISSUE TITLE
nNeoScryptOptions unused -> SSE2 always disabled

### ISSUE THREAD

**Issue by toxeus** [ghostlander/Phoenixcoin-v0.6#9]
I've noticed that `NeoScryptOptions` is never ored into `neoscrypt()`'s profile parameter. Therefore, SSE2 was never enabled.

```sh
~/src/Phoenixcoin ▶ git grep nNeoScryptOptions
src/init.cpp:            nNeoScryptOptions |= 0x1000;
src/util.cpp:uint nNeoScryptOptions = 0;
src/util.h:extern uint nNeoScryptOptions;
```

**Comment by ghostlander** [comment]
Thanks for reporting the bug. Fixed in f329479

**Comment by toxeus** [comment]
You're welcome :+1:
